src/convert.jl

```diff
@@ -103,6 +112,11 @@ function trixi2vtk(filename::AbstractString...;
     verbose && println("| Reading mesh file...")
     @timeit "read mesh" mesh = Trixi.load_mesh_serial(meshfile; n_cells_max=0, RealT=Float64)
 
+    # Check compatibility of the mesh type and the output format
+    if format == :vti && !(mesh isa Trixi.TreeMesh)
+      error("VTI format only available for 2D TreeMesh")
```

Comment:
```suggestion
      throw(ArgumentError("VTI format only available for 2D TreeMesh"))
```

Reply:
What's the advantage? A more specific error message?

Reply:
That's one part. The other aspect is that it allows us to test whether we throw this kind of error with `@test_throws` instead of the manual checking of a specific error message (which we may change).

Reply:
Thanks!

Reply:
Okay, I can update the tests that purposely trigger errors to not rely on the specific wording.